pdns/dnsdist-lua-actions.cc

```diff
@@ -1712,6 +1712,32 @@ class SetTagResponseAction : public DNSResponseAction
   std::string d_value;
 };
 
+class ClearRecordTypesResponseAction : public DNSResponseAction, public boost::noncopyable
+{
+public:
+  ClearRecordTypesResponseAction() {}
+
+  ClearRecordTypesResponseAction(const std::set<QType>& qtypes) : d_qtypes(qtypes)
+  {
+  }
+
+  DNSResponseAction::Action operator()(DNSResponse* dr, std::string* ruleresult) const override
+  {
+    if (d_qtypes.size() > 0) {
+      clearDNSPacketRecordTypes(dr->getMutableData(), d_qtypes);
+    }
+    return DNSResponseAction::Action::HeaderModify;
```

Comment:
I _think_ we might not want to stop the processing of subsequent rules after this one, what do you think? That would mean returning `DNSResponseAction::Action::None` instead., listing it in the list of actions that do not stop the processing in https://dnsdist.org/rules-actions.html#actions (I wonder why that list is broken, btw), and adding `Subsequent rules are processed after this action.` to its description.

Reply:
"Subsequent rules are *not* processed after this action" you mean ?

Reply:
Well it depends :) If we keep returning `DNSResponseAction::Action::HeaderModify` then we do not have to change anything, but if we decide to return `DNSResponseAction::Action::None` instead then `Subsequent rules are processed after this action` would be correct, right?

Reply:
My bad, thanks!